media-api/app/lib/elasticsearch/QueryBuilder.scala

```diff
@@ -33,7 +34,10 @@ class QueryBuilder(matchFields: Seq[String]) {
       case Phrase(value) => matchPhraseQuery(field, value)
       case DateRange(start, end) => rangeQuery(field).from(start.toString()).to(end.toString())
     }
-    case HierarchyField(field, value) => termQuery(field, value)
+    case HierarchyField => condition.value match {
+      case Phrase(value) => termQuery(getFieldPath("pathHierarchy"), value)
+      case _ => throw InvalidQuery("Cannot accept non-Phrase value for HierarchyField Match")
```

Comment:
`case _` should never run because we've set HierarchyField to always have a condition.value that is a `Phrase(value)`.